bentoml/_internal/bento/bento.py

```diff
@@ -93,13 +107,46 @@ def __init__(self, tag: Tag, bento_fs: "FS"):
         self._fs = _Bento_set_fs(self, None, bento_fs)
         self.validate()
 
+    @classmethod
+    def get_download_url(cls, tag: Tag) -> str:
+        yatai_cli = get_current_yatai_cli()
+        bento_version = yatai_cli.presign_bento_version_s3_download_url(tag.name, tag.version)
+        return bento_version.presigned_s3_url
+
     @property
     def tag(self) -> Tag:
         return self._tag
 
-    @staticmethod
+    @classmethod
+    @inject
+    def save_models(cls, bento_fs: "FS", model_tags: t.List[str], model_store: "ModelStore" = Provide[BentoMLContainer.model_store]) -> t.Set[Tag]:
+        models: t.List[Model] = []
+        seen_model_tags: t.Set[Tag] = set()
+        for model_tag in model_tags:
+            try:
+                model_info = model_store.get(model_tag)
+                if model_info.tag not in seen_model_tags:
+                    seen_model_tags.add(model_info.tag)
+                    models.append(model_info)
+            except FileNotFoundError:
+                raise BentoMLException(
+                    f"Model {model_tag} not found in local model store"
+                )
+
+        logger.info(
```

Comment:
this logging message is specifically for the bento build process, it will be confusing showing it in `pull`